### ISSUE TITLE
Unnecessary annotation @RequiresApi(api = Build.VERSION_CODES.M) for IBiometricKeystore

### ISSUE THREAD

**Issue by OndraBasler** [wultra/powerauth-mobile-sdk#525]
I noticed that several functions in PowerAuthSDK have annotation `@RequiresApi(api = Build.VERSION_CODES.M)` because they use `BiometricAuthentication.getBiometricKeystore()`.
It looks like that IBiometricKeystore has different implementation for Android 6+ and prior Android 6. In the snippet below you can see how this is handled.
```
    public static @NonNull IBiometricKeystore getBiometricKeystore() {
        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
            return new BiometricKeystore();
        }
        return new DummyBiometricKeystore();
    }
```
Is the `@RequiresApi(api = Build.VERSION_CODES.M)` annotation really necessary for all methods?

I use Android powerauth SDK version 1.7.8.

**Comment by hvge** [comment]
Yes, you're right. We added `BiometricAuthentication` abstraction in version `1.2.0` but the high level APIs still declare that "M" is required. 

I'll fix this in 1.8.0 release.

**Comment by hvge** [comment]
This can be also fixed by increasing min supported version in SDK to Android 9+ (API level 28) and remove all such annotations from the code. We're thinking about this because the support library we're using (`androidx.biometric`) is in permanent "beta" for a couple of years, so we may switch back to the direct usage of "BiometricPrompt". That would simplify a lot of things.
